### DISCUSSION TITLE
Removing OOC Positions upon Rostering

### DISCUSSION THREAD

**Discussion by dvoraen** [Arx-Game/arxcode#402] [General]
This was brought up on the stories channel today and is one thing that probably needs updated.  If a player is flagged as a Player Guide or Story Coordinator or Storyteller and they're a roster character, then they roster, the OOC status is still listed on the character.  (For an example, see Aethan's +finger as of this writing.)

Should it be removed whenever a Roster Character is put back onto the roster?  The main issue is: suppose a new player takes up that character.  They'll inherit all that OOC status that isn't necessarily true about them (this could matter for being a Senior Player Storyteller, for example).  Staff will then have to manually go in and delete that status from that character, assuming confusion doesn't happen first as the way staff discovers the issue.

**Comment by rozzingit** [comment]
I just want to make @dvoraen feel bad by saying I noted this in #135
